recipe/meta.yaml

```diff
@@ -54,21 +54,3 @@ extra:
   recipe-maintainers:
     - conda-forge/r
     - nsoranzo
-
```

Comment:
@nsoranzo please let those comments in. We keep them for new recipes as some kind of metadata.

Reply:
Many of these metadata will be out-of-date pretty quickly, isn't all these info just a click away?

Reply:
Was not my decision. But I guess ultimately we want also to regenerate the recipes on version update.

Reply:
Either way, its not super important I think.__